unitex/build.xml

```diff
@@ -63,8 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+		<jar jarfile="/Users/rojo/Unitex-GramLab-3.1/App/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
 			<zipgroupfileset dir="${lib-dir}" includes="juniversalchardet-1.0.3.jar"></zipgroupfileset>	
+			<zipgroupfileset dir="${lib-dir}" includes="poi-3.16.jar"></zipgroupfileset>	
```

Comment:
Why this third-party library is requested?

Reply:
See e.g. https://travis-ci.org/UnitexGramLab/gramlab-ide/jobs/277200266#L489

Reply:
This library is required because there are a module which makes statistics with dictionnaries and export it in an excel files